wiki/Ranking_Criteria/Difficulty_Naming/en.md

```diff
@@ -0,0 +1,107 @@
+
+## Default	
+
+This subsection shows the default **scaling** difficulty names for each game mode, which are the most commonly used.
+
+- ![Marathon](/wiki/shared/diff/expert-s.png "Marathon") **Marathon**: a map that is over 5 minutes of [drain time](/wiki/drain_time), and thus can form a mapset on its own.
+
+### ![osu!](/wiki/shared/Osu.gif "osu!") osu!
+ 
+- ![Easy](/wiki/shared/diff/easy-s.png "Easy") Easy 
+- ![Normal](/wiki/shared/diff/normal-s.png "Normal") Normal
+- ![Hard](/wiki/shared/diff/hard-s.png "Hard") Hard
+- ![Insane](/wiki/shared/diff/insane-s.png "Insane") Insane
+- ![Expert](/wiki/shared/diff/expert-s.png "Expert") Expert
+
+### ![osu!taiko](/wiki/shared/Taiko.gif "osu!taiko") osu!taiko
+
+- ![Kantan](/wiki/shared/diff/easy-t.png "Kantan") Kantan
+- ![Futsuu](/wiki/shared/diff/normal-t.png "Futsuu") Futsuu
+- ![Muzukashii](/wiki/shared/diff/hard-t.png "Muzukashii") Muzukashii
+- ![Oni](/wiki/shared/diff/insane-t.png "Oni") Oni
+- ![Inner Oni / Ura Oni](/wiki/shared/diff/expert-t.png "Inner Oni / Ura Oni") Inner Oni / Ura Oni
+
+### ![osu!catch](/wiki/shared/Ctb.gif "osu!catch") osu!catch
+
+- ![Cup](/wiki/shared/diff/easy-c.png "Cup") Cup 
+- ![Salad](/wiki/shared/diff/normal-c.png "Salad") Salad
+- ![Platter](/wiki/shared/diff/hard-c.png "Platter") Platter
+- ![Rain](/wiki/shared/diff/insane-c.png "Rain") Rain
+- ![Overdose](/wiki/shared/diff/expert-c.png "Overdose") Overdose
+
+### ![osu!mania](/wiki/shared/Mania.gif "osu!mania") osu!mania
+
+- ![EZ](/wiki/shared/diff/easy-m.png "EZ") EZ 
+- ![NM](/wiki/shared/diff/normal-m.png "NM") NM
+- ![HD](/wiki/shared/diff/hard-m.png "HD") HD
+- ![MX](/wiki/shared/diff/insane-m.png "NX") MX
+- ![SC](/wiki/shared/diff/expert-m.png "SC") SC
+
+## Songs from other Rhythm Games
+
+This subsection shows the **scaling** difficulty names for mapsets of songs that come from other rhythm games. Using said rhythm game's style of difficulty naming is acceptable.
+
+### Beatmania IIDX (or BMS songs)
+
+- ![Beginner](/wiki/shared/diff/easy-s.png "Beginner") Beginner
+- ![Normal](/wiki/shared/diff/normal-s.png "Normal") Normal
+- ![Hyper](/wiki/shared/diff/hard-s.png "Hyper") Hyper
+- ![Another](/wiki/shared/diff/insane-s.png "Another") Another
+- ![Black Another](/wiki/shared/diff/expert-s.png "Black Another") Black Another
+
+### SOUND VOLTEX (SDVX)
+
+Abbreviations on this particular difficulty naming scheme are acceptable (e.g. NOVICE → NOV).
+
+- ![BASIC](/wiki/shared/diff/easy-s.png "BASIC") BASIC 
+- ![NOVICE](/wiki/shared/diff/normal-s.png "NOVICE") NOVICE
+- ![ADVANCED](/wiki/shared/diff/hard-s.png "ADVANCED") ADVANCED
+- ![EXHAUST](/wiki/shared/diff/insane-s.png "EXHAUST") EXHAUST
+- ![INFINITE](/wiki/shared/diff/expert-s.png "INFINITE") INFINITE
```

Comment:
Might be wrong, but I think INFINITE is interchangeable with "GRAVITY" in the GRAVITY WARS series

Reply:
talked with Feerum about this, every series of SDVX uses their own last diff name, so I think it's not worth adding into the RC. If any guide is made in the future, it can be referenced through this article.